pkg/services/publicdashboards/api/query.go

```diff
@@ -44,6 +44,8 @@ func (api *Api) ViewPublicDashboard(c *models.ReqContext) response.Response {
 		PublicDashboardAccessToken: pubdash.AccessToken,
 		PublicDashboardUID:         pubdash.Uid,
 	}
+	dash.Data.Get("timepicker").Set("hidden", !pubdash.TimeSelectionEnabled)
+	dash.Data.Set("title", "")
```

Comment:
have to set empty title in order not to be shown in the navbar

Reply:
Why not show title? It feels like an important part of the dashboard that gives the data context?